src/tests/unit/recipeService/getRecipe.test.ts

```diff
@@ -0,0 +1,41 @@
+import { IRecipe } from '../../../interfaces/recipe';
+import { Recipe } from '../../../models/recipeModel';
+import RecipeService from '../../../services/recipeService';
+
+import {
+  mockBody,
+  mockDate,
+  mockId,
+  mockImage,
+  mockTitle,
+} from '../../mocks/mockRecipe';
+
+import { setupTests } from '../../setupTests';
+
+const recipeService = new RecipeService();
+
+setupTests('getRecipe', () => {
+  test('gets _id, ownerId, title, imageUrl, body and createdAt of a recipe with specified Id', async () => {
+    const newRecipe = <IRecipe>await new Recipe({
+      ownerId: mockId,
+      title: mockTitle,
+      imageUrl: mockImage,
+      body: mockBody,
+      createdAt: mockDate,
+    }).save();
+    const recipe = await recipeService.getRecipe(newRecipe._id);
+
+    expect(recipe?.ownerId).toBe(mockId);
+    expect(recipe?.title).toBe(mockTitle);
+    expect(recipe?.imageUrl).toBe(mockImage);
+    expect(recipe?.body).toBe(mockBody);
+    expect(recipe?.createdAt).toStrictEqual(mockDate);
+  });
+  test("doesn't return recipe when id doesn't exists", async () => {
+    expect(await recipeService.getRecipe(mockId)).toBeNull();
+  });
+
+  test('no id provided', async () => {
```

Comment:
I would change names of all your tests, i.e. for this test: => "should throw an error when no id is provided",
"doesn't return recipe when id doesn't exists" => "should not return recipe when id does not exist",
"no title provided" => "should throw en error when no title is provided".
But that depends mostly which project you work on. If you feel that these names are ok, you can leave them